nav2_mppi_controller/src/critics/obstacles_critic.cpp

```diff
@@ -158,25 +159,21 @@ void ObstaclesCritic::score(CriticData & data)
       if (!near_goal) {
         repulsive_cost[i] += inflation_radius_ - dist_to_obj;
       }
-    }
```

Comment:
I think there's a mistake here introduced. I think the `}` should be here and the `trajectory_collide` logic back under the parent scope